tasks/simple-batch-job/README.adoc

```diff
@@ -0,0 +1,244 @@
+:sectnums:
+= Batch Job on Cloud Foundry
+
+In this demonstration, you will learn how to orchestrate short-lived data processing application (_eg: Spring Batch Jobs_) using http://cloud.spring.io/spring-cloud-task/[Spring Cloud Task] and http://cloud.spring.io/spring-cloud-dataflow/[Spring Cloud Data Flow] on Cloud Foundry.
+
+== Using Cloud Foundry Server
+
+=== Prerequisites
+
+In order to get started, make sure that you have the following components:
+
+* Local https://pivotal.io/pcf-dev[PCFDev] instance 
+* Local install of https://github.com/cloudfoundry/v3-cli-plugin[V3-CF-CLI] plugin
+* Local build of https://github.com/spring-cloud/spring-cloud-dataflow[Spring Cloud Data Flow]
+* Local build of Spring Cloud Data Flow's https://github.com/spring-cloud/spring-cloud-dataflow-server-cloudfoundry[Cloud Foundry Server]
+* Running instance of mysql in PCFDev
+
+NOTE: PCF 1.7.12 or greater is required to run Tasks on Spring Cloud Data Flow. As of this writing, PCFDev and PWS supports builds upon this version.
+
+=== Running the Sample in Cloud Foundry
+
+. Verify that CF instance is reachable
++
+
+```
+→ cf api
+API endpoint: https://api.local.pcfdev.io (API version: 2.54.0)
+
+$ cf apps
+Getting apps in org user-dataflow / space development as user...
+OK
+
+No apps found
+```
++
+. Follow the http://docs.spring.io/spring-cloud-dataflow-server-cloudfoundry/docs/current-SNAPSHOT/reference/htmlsingle/#getting-started[getting-started] instructions to deploy Spring Cloud Data Flow's Cloud Foundry Server
+
++
+NOTE: For this sample, all you need is the `mysql` service and in PCFDev, the `mysql` service comes with a different plan. From CF CLI, create the service by: `cf create-service p-mysql 512mb mysql` and bind this service to `dataflow-server` by: `cf bind-service dataflow-server mysql`.
```

Comment:
also we need to update the note to say that they need to update SERVICES environment variable to include mysql: 
`cf set-env dataflow-server SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_SERVICES mysql`.  
If you agree I can fix on merge.